index.html

```diff
@@ -582,26 +582,51 @@ <h3 class="ui header">Indices status and management</h3>
     <pre><code>GET /my_index_name/_upgrade?pretty&amp;human</code></pre>
 
 
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
-
+	<div class="es2 es5">
+	<a name="reIndexes"></a>
+	<h2 class="ui dividing header">Reindex API</h2>
+	<h3 class="ui header">Simple Reindex Operation</h3>
+	<pre class="es2 es5"><code>POST /_reindex
+{
+  "source": {
+    "index": "test-index"
+  },
+  "dest": {
+    "index": "test-index-new"
+  }
+}</code></pre>
+	<pre class="es2 es5"><code>POST /_reindex
+{
+  "source": {
+    "index": "test-index"
+  },
+  "dest": {
+    "index": "test-index-new"
+  }
+}</code></pre>
+	<h3 class="ui header">Selective Reindex Operation</h3>
+	<p >
+      we need to separate out the document of female employees and index to another index.
+    </p>
+    <pre class="es2 es5"><code>POST /_reindex
+{
+  "source": {
+    "index": "test-index",
+    "query": {
+      "match": {
+        "gender": "female"
+      }
+    }
+  },
+  "dest": {
+    "index": "test-index-new",
+    "type": "female"
+  }
+}
+}
```

Comment:
Looks like your JSON is not valid here?

Reply:
Yes, it's a typo. I have resolved this issue and submitted the change. please review.